tests/oauth1/rfc5849/test_signatures.py

```diff
@@ -339,7 +359,6 @@ def test_base_string_uri(self):
         self.assertRaises(ValueError, base_string_uri, 'http://:8080')
 
         # Port is not a valid TCP/IP port number
-        self.assertRaises(ValueError, base_string_uri, 'http://eg.com:0')
```

Comment:
Valid port numbers should be <0, 65535>

Reply:
I think this test is valid. Port 0 is a reserved port, it's the default when creating a socket and directs the system to select a "random" port. So specifying a port 0 could cause timeouts and other issues - I'd say failing early here is a good approach.